src/app/external-cluster-wizard/steps/summary/external-cluster-details/external-cluster-details-gke/template.html

```diff
@@ -0,0 +1,51 @@
+<!--
+Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<km-external-cluster-details [cluster]="cluster">
+  <ng-container leftColumn>
+    <km-property>
+      <div key>Zone</div>
+      <div value>
+        <span>{{ cluster.cloud.gke.zone }}</span>
+      </div>
+    </km-property>
+
+    <km-property>
+      <div key>Initial Node Count</div>
+      <div value>
+        <span>{{ spec.initialNodeCount }}</span>
+      </div>
+    </km-property>
+  </ng-container>
+
+  <ng-container rightColumn>
+    <div class="section-header">Kubernetes Version</div>
+
+    <km-property *ngIf="spec.releaseChannel">
```

Comment:
Add *ngIf` for `staticChannel` to show `STATIC` in case of static release channel is selected